### DISCUSSION TITLE
I am not able to reset my password.

### DISCUSSION THREAD

**Discussion by ahmad12367** [community/community#54827] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

I have forgotten my password and when i am trying to reset my password then getting the below issue.

"That address is either invalid, not a verified primary email or is not associated with a personal user account."

Can you please help me?

Regards,
Ahmad

**Comment by ryanlua** [comment]
Are you sure you have the right email? Is there another email connected to the account, if so use that.

**Reply by ahmad12367** [reply]
Hi Ryan,

Yes, i am using the right email address. There is no other email connected to that account.

Regards,
Ahmad

**Reply by gorantladivya1** [reply]
> Are you sure you have the right email? Is there another email connected to the account, if so use that.

Hi Team, I have the right username and right email id but I forgot my password and lost my recovery codes. so want  to reset my password

**Comment by nirav-gajera** [comment, marked as answer]
Check if the email address you are using is verified. Github requires email addresses to be verified before they can be used for password resets. If the email address is not verified, you will need to verify it first.

If not work, you can try contacting Github's support team for further assistance. They will be able to help you troubleshoot the issue and regain access to your account.

**Reply by ahmad12367** [reply]
Hi,

I am trying to verify by logging with other email id and added the email id that need to be verified. Its showing that we send an verification email but that verification email i am not receiving. I have created a ticket to Gihub support team.Let see.

Regards,
Ahnad

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @ahmad12367! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by olatunde-smith** [reply]
am having the same problem as @ahmad12367 . pls help

**Comment by prachi-baisoya09** [comment]
HI, I forget my github password, i am trying to reset it , i solved the puzzle many times, and hit send email for reset button, but i am not getting any email ,moreover getting attached error. please help me in reseting the password , as my token is expired and i need password to regenerate it.
![image](https://github.com/community/community/assets/117898211/3e794bf1-813e-426a-ab7f-18df9e380b45)
![image](https://github.com/community/community/assets/117898211/b126f639-cf9b-4ce0-a9d9-fedb577b1bf0)

**Reply by SudipRoy2024** [reply]
I'm also getting the same issue, not able to resolve it. Anybody, please help

**Reply by prachi-baisoya09** [reply]
try resetting password from mobile phone, it worked for me.

**Reply by SudipRoy2024** [reply]
Thanks @prachi-baisoya09 , that works.

**Comment by beeanadou** [comment]
FYI:
Microsoft edge using my laptop, exactly the same issue above.
Using chrome on my mobile phone, success!

**Comment by abeje-ashenafi** [comment]
https://private-user-images.githubusercontent.com/117898211/309727777-b126f639-cf9b-4ce0-a9d9-fedb577b1bf0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcwOTIxMDYsIm5iZiI6MTcxNzA5MTgwNiwicGF0aCI6Ii8xMTc4OTgyMTEvMzA5NzI3Nzc3LWIxMjZmNjM5LWNmOWItNGNlMC1hOWQ5LWZlZGI1NzdiMWJmMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUzMFQxNzU2NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNDRmNDYwNzkxMDliMTg3MTBlM2YzY2U0NzI5ODY2ZWM4YmZkNDg3ODA0MjNhYjJlMjc2ZTJjMjJjODQyOTIxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.o1FQYKZpKu7KPZ0xE0OiNwcRyquIlW5zXoWwhXmvrlc

**Comment by jamesmulero** [comment]
I have this issue but I can remember my username, email address and password. I left the organisation whose email address I used so I cannot receive the password reset link. What can I do in this case, please?

**Comment by putrenkov** [comment]
Have same problem Could not reset my account by email I do not get letter!!!!!!!!!!!!
Could get my account and find place were someone could help!

**Reply by ghost** [reply]
Did you find any answer?

**Reply by lMysticl** [reply]
> answer?

Yes, I wrote support they help

**Comment by Prasad-Nov24** [comment]
I am facing same issue with my organization email. I am not getting password reset mail, since it is saying "This address is either invalid, not a verified primary email or is not associated with a personal user account. Organization billing emails are only for notifications." Please help me to resolve this.

**Comment by ksoni-splunk1** [comment]
same here, can someone please help?

**Comment by IbrahimWad** [comment]
I have try to reset my password by my username and it work in my case

**Comment by jinwater88** [comment]
I also encountered the situation where GitHub could not reset the password. I only remembered the email address but not the username. I reset the password via email and the following situation occurred，
However, I used the email address to register a new account and reset the password, but the previously registered account cannot be retrieved.

**Comment by ThanhNVQ** [comment]
My account has blocked because i try to login many time with failure password . But I click button forgot password and has done capcha, i don't see screen "Send Email Recovery Password". Sometimes, I get notification that "Too many attempts. Please wait a while and try again."
Please help me to recovery account.
![72a13d2b-f2d9-4514-90d3-80b17ba14dd9](https://github.com/user-attachments/assets/d4735610-e537-47a0-b3b8-fed5fa6818ec)

**Comment by mukeshgoyalchem** [comment]
<img width="1152" height="648" alt="hhhh" src="https://github.com/user-attachments/assets/86008832-aa0f-4a2b-a882-2d2c4dd0bb6d" />
I am not able to forgot or not able to sign in my account so i request you to provide access for my account 
mail :- ramajee.singh@techsciresearch.com

i  dont have verify profile mail get first time so i request you to provide me complete solution on this
